lib/java/kafka/schema/src/main/java/co/airy/kafka/schema/application/ApplicationCommunicationContacts.java

```diff
@@ -0,0 +1,17 @@
+package co.airy.kafka.schema.application;
+
+import co.airy.kafka.schema.ApplicationCommunication;
+
+import java.util.Map;
+
+public class ApplicationCommunicationContacts extends ApplicationCommunication {
```

Comment:
In my opinion, it makes sense to store all contact-related metadata in a separate topic. Even though the data structure is the same the data has a different meaning and we should be able to separate it on a storage level.